readme.md

```diff
@@ -250,6 +250,12 @@ Enforce ES2015+ rules. Disabling this will make it not *enforce* ES2015+ syntax
 
 *ES2015+ is parsed even without this option. You can already use ES2017 features like [`async`/`await`](https://github.com/lukehoban/ecmascript-asyncawait).
 
+### webpack
+
+Type: `boolean | object`
+
+Use [eslint-import-resolver-webpack](https://github.com/benmosher/eslint-plugin-import) to resolve import search paths. Value can be an object containing options that should be passed to the resolver.
```

Comment:
```suggestion
Use [`eslint-import-resolver-webpack`](https://github.com/benmosher/eslint-plugin-import) to resolve import search paths. The value can be an object containing options that should be passed to the resolver.
```

Reply:
The description needs to describe what it does instead of just linking to the `eslint-import-resolver-webpack`. The boolean value is also not described.

Reply:
> object containing options that

`options` here should be linkified.